warehouse/integrations/postgres-legacy/postgres.go

```diff
@@ -529,7 +529,7 @@ func (pg *Postgres) DeleteBy(tableNames []string, params warehouseutils.DeleteBy
 			tb,
 		)
 		pg.logger.Infof("PG: Deleting rows in table in postgres for PG:%s", pg.Warehouse.Destination.ID)
-		pg.logger.Debugf("PG: Executing the statement  %v", sqlStatement)
+		pg.logger.Infof("PG: Executing the statement  %v", sqlStatement)
```

Comment:
Did we change this on purpose?

Reply:
No this was during debugging... I will revert this back to original